src/modules/sync_team_label_with_project/index.ts

```diff
@@ -2,7 +2,7 @@ import {
   AI_EXPERIMENTS, AZURE,
   BAAS,
   BILLING,
-  DBAAS,
+  DBAAS, FE_INFRA,
```

Comment:
Probably format this better? :D 
(Should I add a CI formatting job here? :D)

Reply:
🔥(this is fine)>🐶🔥🔥